rnd/autogpt_server/autogpt_server/blocks/rss-reader-block.py

```diff
@@ -32,6 +32,7 @@ class Input(BlockSchema):
         polling_rate: int = SchemaField(
             description="The number of seconds to wait between polling attempts.",
             placeholder="300",
+            gt=0,
```

Comment:
Is this intentional? If so, maybe set it a bit higher so we don't run into insane server loads from this later on.

Reply:
Reverted, I think we should have a PR to set all this or guidelines for the schema so invalid inputs are caught before triggering backend.